examples/gaussian.py

```diff
@@ -15,6 +15,7 @@
 model = A*exp(-((x - x0)**2/(2 * sig**2)))
 
 # Sample 10000 points from a N(15.0, 1.5) distrubution
+np.random.seed(seed=123456789)
```

Comment:
#120 has been solved on master, does that also solve the Heissenbug nature of this one to some extend? (I tried to form a new release, but that gave a new bug unfortunately so stay tuned.)